app/views/layouts/application.html.erb

```diff
@@ -10,5 +10,11 @@
 
   <body>
     <%= yield %>
+
+    <header class="topnav">
+      <h2> Fruits Galore</h2>
+      <a href="/merchants">Merchants</a>
+      <a href="/items">Items</a>
+    </header>
   </body>
 </html>
```

Comment:
Line 12 is where all of your views would get yielded to. If you want this topnav to be at the top, you'll have to bring this header element before the yield. This may be why you went ahead and wrapped all of your views in html and body tags 🤷‍♀